<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogoption</class>
 <widget class="QDialog" name="Dialogoption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>خيارات</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/image/groupb.png</normaloff>:/images/image/groupb.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>خيارات عامة</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>عدد الكتب السابقة :</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_3">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>15</number>
           </property>
           <property name="value">
            <number>7</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>مسارالكتب :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_costmPath">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_costmPath">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>اللغة :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxLng"/>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxLngLayou">
           <property name="text">
            <string>الاتجاه من اليسار الى اليمين</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>الأيقونات وشريط الأدوات</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="radioButtonStandardTool">
            <property name="toolTip">
             <string>سيتم تطبيق التغيير بعد اعادة تشغيل البرنامج</string>
            </property>
            <property name="text">
             <string>عرض القائمة وشريط الادوات الافتراضي</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonRubon">
            <property name="toolTip">
             <string>سيتم تطبيق التغيير بعد اعادة تشغيل البرنامج</string>
            </property>
            <property name="text">
             <string>شريط ادوات مدمج مع القائمة</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QCheckBox" name="checkBoxTxtUnderIcon">
              <property name="text">
               <string>نص تحت الايقونات</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_15">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>حجم الايقونة :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxIconSize">
              <property name="minimum">
               <number>16</number>
              </property>
              <property name="maximum">
               <number>48</number>
              </property>
              <property name="singleStep">
               <number>2</number>
              </property>
              <property name="value">
               <number>22</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>السمة</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>السمة العامة :</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QCheckBox" name="checkBoxStyleCostum">
              <property name="text">
               <string>سمة مخصصة</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditStyleCostum">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonStyleCostum">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="checkBoxCadre">
              <property name="text">
               <string>عرض إطار النص</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxCadre">
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>194</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string> الخطوط</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>خط الصفحة</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_font">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="minimumSize">
            <size>
             <width>46</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string>...اختر</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>خط العناين</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_fontTitleSize">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_fontTitle">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonTitle">
           <property name="minimumSize">
            <size>
             <width>46</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string>...اختر</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>خط الحاشية</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_fontHachia">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonHachia">
           <property name="minimumSize">
            <size>
             <width>46</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string>...اختر</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_37">
         <item>
          <widget class="QLabel" name="label_35">
           <property name="text">
            <string>خط مخصص</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_28">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_fontKhasse">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonkhass">
           <property name="minimumSize">
            <size>
             <width>46</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string>...اختر</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>استخدم ارقاما</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxNumber">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>الالوان</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>الالوان المخصصة</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_prefercolor">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>لون خط الصفحة</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_fontcolor">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>لون العناوين</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_titlecolor">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>لون تمييز البحث</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>341</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_findcolor">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_36">
         <item>
          <widget class="QLabel" name="label_34">
           <property name="text">
            <string>اون الخلفية</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_27">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>97</width>
             <height>24</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_backcolor">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxPdfColor">
         <property name="text">
          <string>استخدم لون الخلفية لملفات pdf</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialogoption</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxCadre</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxCadre</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxStyleCostum</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxStyleCostum</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_9</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxStyleCostum</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditStyleCostum</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxStyleCostum</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButtonStyleCostum</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
